api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSource.java

```diff
@@ -28,6 +28,8 @@
  *******************************************************************************/
 package org.eclipse.microprofile.config.spi;
 
+import java.util.HashMap;
+import java.util.LinkedHashMap;
```

Comment:
Don't think the `LinkedHashMap` import is used

Reply:
removed.